californium-proxy/pom.xml

```diff
@@ -18,6 +18,7 @@
 
 	<properties>
 		<httpcomponents.version>4.2.5</httpcomponents.version>
+		<httpasyncclient.version>4.1.3</httpasyncclient.version>
```

Comment:
in order for this to work properly we should also update http-core etc accordingly. IMHO since the dependency on the async client pulls in everything else needed from  HttpComponents, we should remove the explicit dependency on httpcore-nio and also remove the `httpcomponents.version` property.